src/visitable.cpp

```diff
@@ -164,7 +148,8 @@ VisitResponse visitable<vehicle_cursor>::visit_items( const std::function<VisitR
 }
 
 template <>
-VisitResponse visitable<vehicle_selector>::visit_items( const std::function<VisitResponse( item *, item *, const tripoint* )>& func )
+VisitResponse visitable<vehicle_selector>::visit_items(
```

Comment:
We need the second argument to implement `find_parent` and also algorithms such as `remove_item` will require a parent pointer. The third argument is currently unused but is effectively free to provide and can be useful for example with selectors to determine which an `item` is on:

```
map_selector sel( pos );
sel.visit_items( item *it, item *parent, const tripoint *where ) {
   g->m.i_at( *where ) ... // where am I?
}
```